src/irc/ClientPool.ts

```diff
@@ -289,7 +291,8 @@ export class ClientPool {
 
         return new BridgedClient(
             server, ircClientConfig, matrixUser || undefined, isBot,
-            this.ircEventBroker, this.identGenerator, this.ipv6Generator
+            this.ircEventBroker, this.identGenerator, this.ipv6Generator,
+            this.encodingFallback
```

Comment:
Would prefer it if we just did:

`this.ircBridge.config.ircService.encodingFallback` rather than copying it to ClientPool

Reply:
That's right. Fixed now.